lib/web_ui/lib/src/engine/html/platform_view.dart

```diff
@@ -75,18 +75,28 @@ class PersistedPlatformView extends PersistedLeafSurface {
     }
   }
 
+  // Platform Views can only be updated if their viewId matches.
+  @override
+  bool canUpdateAsMatch(PersistedSurface oldSurface) {
+    if (super.canUpdateAsMatch(oldSurface)) {
```

Comment:
I wonder if the `PersistedSurface.canUpdateAsMatch` should be made `@mustCallSuper`.

Reply:
Good point, let me check how many other overrides of this we have and see what've been doing there.

Reply:
My platform_view code seemed like the only that was overriding this method. I think calling super here is beneficial, so I've added @mustCallSuper to the base method.